src/PowerShellEditorServices.VSCode/Cmdlets/VSCodeHtmlContentViewCommands.cs

```diff
@@ -125,7 +153,21 @@ public class CloseVSCodeHtmlContentViewCommand : PSCmdlet
         ///
         protected override void BeginProcessing()
         {
-            HtmlContentView.Close().GetAwaiter().GetResult();
+            try
+            {
+                HtmlContentView.Close().GetAwaiter().GetResult();
+            }
+            catch (Exception e)
+            {
+                WriteError(
+                    new ErrorRecord(
+                        e,
+                        "HtmlContentViewCouldNotClose",
+                        ErrorCategory.CloseError,
+                        targetObject: null));
+
+                return;
```

Comment:
The `return` here looks like it might be unneeded

Reply:
yep!